DependencyInjection/Configuration.php

```diff
@@ -83,6 +83,9 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->booleanNode('disable_default_metrics')
                     ->defaultValue(false)
                 ->end()
+                ->booleanNode('disable_default_prometheus_metrics')
```

Comment:
I think, this could be confusing considering the previous var (`disable_default_metrics`). I suggest we rename it to `disable_default_promphp_metrics`.